src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "../Introduce/Introduce.scss";
+import { Link } from "react-router-dom";
+
+const Introduce = () => {
+  function handleInput() {
+    console.log("input 태그동작");
+  }
+
+  function handleButton() {
+    console.log("button 태그동작");
+  }
+
+  return (
+    <div className="box">
+      <div className="boxWwrap">
+        <div className="weCodeBox">
+          <h2>{">"}wecode</h2>
+        </div>
+      </div>
+      <strong className="info">홍태훈</strong>
+      <div className="info">43기</div>
+      <div className="inputBox">
+        <input className="input" onChange={handleInput}></input>
+
+        <Link to="/product">
+          <button className="button" onClick={handleButton}>
+            클릭
+          </button>
+        </Link>
```

Comment:
```suggestion
          <button className="button" onClick={handleButton}>
            클릭
          </button>
```

Link 컴포넌트는 제거하셔도 됩니다!
handleButton 함수에 useNavigate hook을 사용한 페이지 이동을 구현해주세요!